packages/backend/src/server.test.ts

```diff
@@ -472,3 +510,13 @@ async function createFakeDidResolver(correctDids: Record<string, any>) {
     },
   }
 }
+
+async function generateNewInvitation(submissionService: SubmissionService) {
```

Comment:
I wonder why it's in tests? 🤔 I would imagine that it should be part of our business logic.